src/forecast_clarify/model_registry.py

```diff
@@ -0,0 +1,45 @@
+import wandb
```

Comment:
do we need to include this in the requirements?

Reply:
great catch, I'll add it

Reply:
fixed.